### PR TITLE
Add <Prompt> full examples

### PR DESCRIPTION
**Pull request by bld010**

### CODE REVIEW

File: twiml/voice/pay/prompt/full-example-ach/output/full-example-ach.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	<Pay input="dtmf" timeout="5" maxAttempts="3" paymentMethod="ach-debit" language="en-US">
+		<Prompt for="bank-routing-number">
+			<Say>
+				Please enter your bank routing number.
+			</Say>
+		</Prompt>
+
+		<Prompt for="bank-routing-number" errorType= "timeout">
+			<Say>
+			I am sorry I didn't hear anything. Please enter your bank routing number.
```
Comment (thinkingserious) [inline twiml/voice/pay/prompt/full-example-ach/output/full-example-ach.xml:12]: Probably another issue with the generator. This is the only case where the text is not indented.

File: twiml/voice/pay/prompt/full-example-credit-card/full-example-credit-card.3.x.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+});
+const prompt = pay.prompt({
+    for: 'payment-card-number'
+});
+prompt.say('Please enter your credit card number.');
+const prompt2 = pay.prompt({
+    for: 'payment-card-number',
+    errorType: 'timeout'
+});
+prompt2.say('I am sorry I didn\'t hear anything. Please enter your credit card number.');
```
Comment (thinkingserious) [inline twiml/voice/pay/prompt/full-example-credit-card/full-example-credit-card.3.x.js:16]: nit: I think there should be a comma after "I am sorry", or "I am sorry" can be removed. (note that this should be a find/replace to catch other instances if you decide to modify this.

File: twiml/voice/pay/prompt/full-example-credit-card/full-example-credit-card.5.x.cs [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+{
+    static void Main()
+    {
+        var response = new VoiceResponse();
+        var pay = new Pay(paymentMethod: Pay.PaymentMethodEnum.CreditCard, validCardTypes: new []{Pay.ValidCardTypesEnum.Visa, Pay.ValidCardTypesEnum.Mastercard, Pay.ValidCardTypesEnum.Amex}.ToList());
+        var prompt = new Prompt(for_: Prompt.ForEnum.PaymentCardNumber);
+        prompt.Say("Please enter your credit card number.");
+        pay.Append(prompt);
+        var prompt2 = new Prompt(errorType: new []{Prompt.ErrorTypeEnum.Timeout}.ToList(), for_: Prompt.ForEnum.PaymentCardNumber);
+        prompt2.Say("I am sorry I didn't hear anything. Please enter your credit card number.");
```
Comment (thinkingserious) [inline twiml/voice/pay/prompt/full-example-credit-card/full-example-credit-card.5.x.cs:16]: Do we need to escape the ' in this file? Same issue in the .rb and .java file.

File: twiml/voice/pay/prompt/full-example-credit-card/output/full-example-credit-card.xml [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+        I am sorry I didn't hear anything. 
+        Please enter your 5 digit billing zip code.
+      </Say>
+    </Prompt>
+
+    <!-- Prompt after customer has entered all payment information -->
+
+    <Prompt for="payment-processing">
+      <Say>
+        Thank you, please wait while we process your payment
```
Comment (thinkingserious) [inline twiml/voice/pay/prompt/full-example-credit-card/output/full-example-credit-card.xml:127]: Missing trailing period.

**Review by thinkingserious — DISMISSED** [review]

File: twiml/voice/pay/prompt/full-example-credit-card/full-example-credit-card.3.x.js (hunk 2 above)
Comment (bld010) [reply twiml/voice/pay/prompt/full-example-credit-card/full-example-credit-card.3.x.js:16]: Good point!

File: twiml/voice/pay/prompt/full-example-credit-card/output/full-example-credit-card.xml (hunk 4 above)
Comment (bld010) [reply twiml/voice/pay/prompt/full-example-credit-card/output/full-example-credit-card.xml:127]: Accidentally included the wrong versions of the .xml files. Will fix!

**Comment by bld010** [comment]
@thinkingserious Hopefully fixed the issues. Can you take another look?

**Review by thinkingserious — APPROVED** [review]
🎆

**Pull request merged**